packages/integration-tests/src/client/index.ts

```diff
@@ -84,7 +84,7 @@ export default class MockClient {
     assert(this.interactionCookie, new Error('Get cookie from authorization endpoint failed'));
   }
 
-  public async processSession(redirectTo: string) {
+  public async processSession(redirectTo: string, consent = true) {
```

Comment:
better to add some comments about the parameters

Reply:
added